woocommerce/woocommerce
# Fixed `@woocommerce/block-library` Build Dependencies & Output

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

As a consequence of switching to PNPM, some of the dependencies that were being included transitively need to be included directly in order to prevent build failures. This PR adds those dependencies and adjusts the Webpack config so that it builds into the WooCommerce directory.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Run `pnpm --filter='@woocommerce/block-library' build` locally and confirm that it builds and goes to `plugins/woocommerce/assets/client/blocks`.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
